src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -231,6 +246,9 @@ private static JsonObject createRequestContext(Request request) {
       .map(Request::getPickupServicePoint)
       .map(ServicePoint::getName)
       .ifPresent(value -> requestContext.put("servicePointPickup", value));
+    optionalRequest
+      .map(Request::getRequestDate)
+      .ifPresent(value -> write(requestContext, "requestDate", value));
```

Comment:
I believe this should a part of another PR because it's not required in CIRC-1785, but anyway, is it covered with tests?